board.py

```diff
@@ -121,11 +204,22 @@ def row_6(self):
         return buffer
 
     def row_7(self):
+        card_number = 1
+        rank = self.cards[0].get_rank()
+        if rank == 1:
+            card_number = 13
+        elif rank == 2:
+            card_number = 9
+        elif rank == 3:
+            card_number = 5
```

Comment:
should not have a rank == 3, and all numbers should be 4 lower than they are now.